socorro/siglists/README.md

```diff
@@ -0,0 +1,3 @@
+# Signatures Utilities Lists
+
+Great documentation comes here.
```

Comment:
I'm using Markdown about everywhere else, so it's a reflex for me to go with it. I find it a lot easier to write than reST. But you're right, we're using reST in Socorro, so I should stick with it.